content/zh-cn/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -132,7 +132,8 @@ The following sysctls are supported in the _safe_ set:
 - `net.ipv4.ip_local_port_range`,
 - `net.ipv4.tcp_syncookies`,
 - `net.ipv4.ping_group_range`（从 Kubernetes 1.18 开始）,
-- `net.ipv4.ip_unprivileged_port_start`（从 Kubernetes 1.22 开始）。
+- `net.ipv4.ip_unprivileged_port_start`（从 Kubernetes 1.22 开始）,
+- `net.ipv4.ip_local_reserved_ports`（从 Kubernetes 1.27 开始）。
```

Comment:
ditto

Reply:
Hi @hunshcn, you could add EN comments of this list before the translations, like [Line 114-119](https://github.com/kubernetes/website/pull/44267/files#diff-136e7f71d742f538cfb2b838af906f4dae006cc3887e84257dbeb299a10fd9cfR114-R119).

Reply:
done